services/api/src/utils/logging.js

```diff
@@ -1,7 +1,10 @@
 const pino = require('pino');
+const bytes = require('bytes');
 const config = require('@bedrockio/config');
 const { floor } = require('lodash');
 const { getTracer } = require('@bedrockio/instrumentation');
+const dateFormat = require('dateformat');
+const chalk = require('chalk');
```

Comment:
This will is duplicated, the @bedrockio/instrumentation will take that over, to make the logging more portable.
Don't need to copy paste this file around.

Reply:
Which one is that? This is to get local ISO time in the log output via `prettyPrint` config... `pino-pretty` has it but I can't get it how I want it because the config is too obtuse.

If I can get that somehow from instrumentation then great I'll use that instead.